### ISSUE TITLE
Migrar registros sin lat/lng y asignar basados en dirección

### ISSUE THREAD

**Issue by chubas** [azubieta/hgathering-api#5]
Se puede obtener lat/lng usando los servicios de geolocation de google asumiendo que la dirección está bien.
Sería bueno migrar estos datos en el backend para que no se delegue esto al frontend ya que es muy ineficiente

`http://maps.googleapis.com/maps/api/geocode/json?sensor=false&address=' + encodeURIComponent(this.address)`

**Comment by pedrogk** [comment]
Vamos a correr un script para que directamente actualice la colección de mongodb y le agregue las coordenadas.

Irving González (@kodamirmo) está trabajando en eso.

**Comment by chubas** [comment]
👍

**Comment by azubieta** [comment]
Recomiendo que usen la api en lugar de escribir directo en la bd. Para que
el sistema valide los datos y haga bien las relaciones. Y de paso evitar
datos corruptos.

**Comment by pedrogk** [comment]
Sí. Irving está haciendo el script de js y ya con ese veo que lo usemos con la api.

**Comment by kodamirmo** [comment]
Donde esta la documentación de la API?

**Comment by pedrogk** [comment]
@kodamirmo en http://ec2-54-242-119-209.compute-1.amazonaws.com/explorer

**Comment by pedrogk** [comment]
En https://hapi.balterbyte.com/explorer
